api/py/test/test_compile.py

```diff
@@ -79,11 +79,37 @@ def test_basic_compile():
         extract_and_convert, ["--chronon_root=test/sample", "--input_path=joins/sample_team/sample_join.py"]
     )
     assert result.exit_code == 0
+
+
+def test_compile_deprecation():
```

Comment:
Highly recommend keeping the 2 test cases in separate test methods. So each test is testing only 1 use case. Easier to debug if test fails

Reply:
updated